CONTRIBUTING.md

```diff
@@ -65,3 +65,74 @@ Explain the problem and include additional details to help maintainers reproduce
 
 
 This text has been largely inspired by the Atom contributing notes, which amazingly complete
```

Comment:
Maybe you want to fix "which amazingly complete" as well (should probably be "which are amazingly complete").